webpack.config.js

```diff
@@ -5,13 +5,26 @@
 const path = require('path');
 
 module.exports = {
-  entry: './src/index.js',
-  output: {
-    filename: 'glean.js',
-    path: path.resolve(__dirname, 'dist'),
-    library: 'Glean',
-    // This config makes the Glean object a global `var`.
-    // See: https://webpack.js.org/guides/author-libraries/
-    libraryTarget: 'var'
-  }
+    entry: './src/index.js',
+    output: {
+        filename: 'glean.js',
+        path: path.resolve(__dirname, 'dist'),
+        library: 'Glean',
+        // This config makes the Glean object a global `var`.
+        // See: https://webpack.js.org/guides/author-libraries/
+        libraryTarget: 'var'
+    },
+    optimization: {
```

Comment:
Why is this necessary?

Reply:
Sorry to not point that out.  It's not necessary, it just helped have understandable tracebacks (since it doesn't seem like QML does any source mapping).  Useful in development, but we wouldn't do it in production...